src/includes/traits/Plugin/AdminBarUtils.php

```diff
@@ -110,15 +106,31 @@ public function adminBarMenu(\WP_Admin_Bar &$wp_admin_bar)
                 ]
             );
         }
+        /*[/pro]*/
         if ($this->adminBarShowing('cache_clear')) {
+            if (!IS_PRO) {
```

Comment:
@renzms This new `$wp_admin_bar->add_menu()` section is not necessary, as it's already included further below. I suggest removing this `if(!IS_PRO)` statement and simply using [the one below](https://github.com/websharks/comet-cache-pro/blob/2c0ee9b27e56a054629e69109a3c28c4efaf6a29/src/includes/traits/Plugin/AdminBarUtils.php#L187). (You'll need to adjust the `/*[pro strip-from="lite"]*/` markers so that the one below shows up in the Lite version.)